diff -Naur old/nova/policies/__init__.py new/nova/policies/__init__.py
--- old/nova/policies/__init__.py	2017-10-31 09:51:16.418876840 +0000
+++ new/nova/policies/__init__.py	2017-10-31 09:53:55.027078995 +0000
@@ -29,6 +29,7 @@
 from nova.policies import console_output
 from nova.policies import consoles
 from nova.policies import create_backup
+from nova.policies import cloudlet
 from nova.policies import deferred_delete
 from nova.policies import evacuate
 from nova.policies import extended_availability_zone
@@ -88,7 +89,6 @@
 from nova.policies import volumes
 from nova.policies import volumes_attachments

-
 def list_rules():
     return itertools.chain(
         admin_actions.list_rules(),
@@ -107,6 +107,7 @@
         console_output.list_rules(),
         consoles.list_rules(),
         create_backup.list_rules(),
+        cloudlet.list_rules(),
         deferred_delete.list_rules(),
         evacuate.list_rules(),
         extended_availability_zone.list_rules(),